code/modules/events/shuttle_loan.dm

```diff
@@ -165,6 +170,15 @@
 				for(var/i in 1 to 5)
 					var/decal = pick(/obj/effect/decal/cleanable/flour, /obj/effect/decal/cleanable/robot_debris, /obj/effect/decal/cleanable/oil)
 					new decal(pick_n_take(empty_shuttle_turfs))
+			if(PIZZA_DELIVERY)
+				shuttle_spawns.Add(/obj/item/pizzabox/margherita)
+				shuttle_spawns.Add(/obj/item/pizzabox/meat)
+				shuttle_spawns.Add(/obj/item/pizzabox/vegetable)
+				shuttle_spawns.Add(/obj/item/pizzabox/margherita)
+				shuttle_spawns.Add(/obj/item/pizzabox/margherita)
+				shuttle_spawns.Add(/obj/item/pizzabox/meat)
+				if(prob(10))
+					shuttle_spawns.Add(/obj/item/pizzabox/bomb)
```

Comment:
Small suggestion:  add an else to this prob, that way people can't meta "There's seven boxes!  One of them's a bomb!"

Also, for readability's sake, add identical pizza types in adjacent lines.